shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/domain.usecase/LogInUseCaseTest.kt

```diff
@@ -0,0 +1,77 @@
+package co.nimblehq.avishek.phong.kmmic.domain.usecase
+
+import co.nimblehq.avishek.phong.kmmic.domain.model.Token
+import co.nimblehq.avishek.phong.kmmic.domain.repository.AuthenticationRepository
+import io.kotest.matchers.shouldBe
+import io.mockative.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.flow.collect
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+
+@ExperimentalCoroutinesApi
+class LogInUseCaseTest {
+
+    @Mock
+    private val mockRepository = mock(classOf<AuthenticationRepository>())
+
+    @Mock
+    private val testMockToken = mock(classOf<Token>())
+
+    private lateinit var useCase: LogInUseCase
+
+    private val mockThrowable = Throwable("mock")
+    private val mockToken = Token(
+        "",
+        "",
+        0,
+        "",
+        0
+    )
```

Comment:
[Optional] We can make them in one line 🙏
```suggestion
    private val mockToken = Token("", "", 0, "", 0)
```